dashboard/app/models/resource.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: resources
+#
+#  id         :integer          not null, primary key
+#  name       :string(255)
+#  url        :string(255)
+#  embed_slug :string(255)
```

Comment:
could the embed_slug just be based on the key (see above comment). In which case can we just use that instead of having both?

Reply:
I think this is resolved by making `embed_slug` key but let me know if I misunderstood!

Reply:
yup!